File: Beginner/1043 - Triangle/1043.c
#include <stdio.h>
#include <math.h>

int main(){

    double A, B, C;

    scanf("%lf %lf %lf", &A, &B, &C);
    if(A<B+C && B<A+C && C<A+B)
        printf("Perimetro = %.1lf\n", A+B+C);

    else
        printf("Area = %.1lf\n",((A+B)*C)/2);

    return 0;
}

File: Beginner/1043 - Triangle/1043.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        string[] input = Console.ReadLine().Split(' ');
        double A = double.Parse(input[0]);
        double B = double.Parse(input[1]);
        double C = double.Parse(input[2]);

        if(A<B+C && B<A+C && C<A+B)
            Console.WriteLine($"Perimetro = {A+B+C:0.0}");
        else
            Console.WriteLine($"Area = {(A+B)*C/2:0.0}");
    }
}